plugins/chain_api_plugin/chain.swagger.yaml

```diff
@@ -0,0 +1,786 @@
+openapi: 3.0.0
+info:
+  title: Chain API
+  description: "OAS 3.0 Nodeos [chain_api_plugin](https://eosio.github.io/eos/latest/nodeos/plugins/chain_api_plugin/index) API Specification\r"
+  version: 1.0.0
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  contact:
+    url: https://eos.io
+tags: []
+servers:
```

Comment:
Yeah, wildcard servers are great. However, they are still under an assumption of a centralized service, so they cannot (at least through specification and available tooling) be used in a setting where host and port are a complete wildcard. That said it could be enhanced via custom development to allow an end user to input a desired **host** and **path**, but still a shame it wasn't a consideration during its development. 

Ideally the specification would suggest that through user input the host/port could be modified. This would encourage tooling (eventually) provide this feature to its users.